resources/keb/Chart.yaml

```diff
@@ -3,5 +3,5 @@ appVersion: "1.8.34"
 name: keb
 description:
 description: Kyma Environment Broker Helm chart for Kubernetes
-version: 1.8.34
+version: 1.8.35
```

Comment:
Please revert this change. This value is auto bumped by our release workflow.

Reply:
chart version reverted